test/matchers.setup.ts

```diff
@@ -0,0 +1,14 @@
+import { toEqualArrayUnsorted } from "#test/test-utils/matchers/to-equal-array-unsorted";
+import { toHaveTypes } from "#test/test-utils/matchers/to-have-types";
+import { expect } from "vitest";
+
+/**
+ * @module
```

Comment:
```suggestion
/*
```
`@module` is not valid TSDoc, and even if it was Typedoc doesn't parse the `test/` directory.